src/foam/u2/filter/properties/BooleanFilterView.js

```diff
@@ -66,18 +65,9 @@ foam.CLASS({
       required: true
     },
     {
-      class: 'Boolean',
-      name: 'boolT',
-      label: 'True',
-      documentation: 'Filter property for True, mostly for UI/UX purposes',
-      value: false
-    },
-    {
-      class: 'Boolean',
-      name: 'boolF',
-      label: 'False',
-      documentation: 'Filter property for False, mostly for UI/UX purposes',
-      value: false
+      class: 'Int',
```

Comment:
Does this mean I can no longer bind it directly to a boolean property? Couldn't you use three values as true, false and undefined, which FOAM supports?

Reply:
Completely overlooked; I agree completely.

Reply:
I wasn't able to get this working. The adapt function on Boolean converts `undefined` to `false`. If we want to use these value we would need a `Ternary` property that allows undefined.